src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -170,8 +314,51 @@ You can always lookup an app's current locale with
 Locale myLocale = Localizations.localeOf(context);
 ```
 
+<a name="specifying-supportedlocales"></a>
+### Specifying the app's supported&shy;Locales parameter
+
+Although Flutter's flutter_localizations library includes support
```

Comment:
=> Although the `flutter_localizations` library currently supports 77
      languages and language variants, only....